### PR TITLE
update metadata

### PR DESCRIPTION
**Pull request by projectmushroom**
- [x] update meta description + keywords
- [x] update Open Graph objects
- [x] update image sizes + fallback image

### CODE REVIEW

File: iogt/templates/base.html [hunk 1]
```diff
@@ -8,21 +8,19 @@
         <meta charset="utf-8" />
         <title>{% block title %}{% if self.seo_title %}{{ self.seo_title }}{% else %}{{ self.title }}{% endif %}{% endblock %}{% block title_suffix %}{% endblock %}</title>
         <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-        <meta name="description" content="" />
-        <meta name="keywords" content="" />
-
+        <meta name="description" content="{% if self.social_media_description %}{{ self.social_media_description }}{% else %}{{ self.subtitle }}{% endif %}" />
```
Comment (Mitso) [inline iogt/templates/base.html:11]: This works for Sections and Article pages but not the homepage I think - A static text option is a good idea.

**Review by Mitso — APPROVED** [review]
Left a comment otherwise looks good.

File: iogt/templates/base.html (hunk 1 above)
Comment (projectmushroom) [reply iogt/templates/base.html:11]: yes, I'm requesting a default description from them, will update before landing

Comment (projectmushroom) [reply iogt/templates/base.html:11]: @Mitso default text + translations to be added in a separate ticket

**Pull request merged**